run.js

```diff
@@ -68,16 +75,28 @@ async function runUpdate() {
     });
 
     parser.on('end', async () => {
-      console.log('RECORDS: ', records);
-      console.log('INVALID RECORDS: ', invalidRecords);
+      if (records.length) {
+        logger.log('info', 'Starting record insertion to temp table');
+        await db.insertRecords(client, records);
+        logger.log('info', 'Replacing main lookup table from new inserts');
+        await db.replaceLookupTable(client);
+      }
       logger.log('info', 'Job complete!');
+      console.log('INVALID RECORDS: ', invalidRecords);
```

Comment:
Good idea. I will probably remove the actual logging of the invalid records in the next ticket (invalid records will be added to notifications, but want to avoid logging user data where possible), but keep this conditional logging of the warning.